notifications/06-24-1100.md

```diff
@@ -0,0 +1,32 @@
+**What is happening?**
+
+ITMS RFC 195614
+
+CITZ - PROD MCS - INC0032789 - Apply suggested config change to sysdig agents
+
+**When?**
+
+Starting this afternoon (Wednesday June 24th) at 3:00pm PDT and is expected to last for approximately 15 minutes.
```

Comment:
I've found that it takes more than 15 min (maybe an hour?) for the DaemonSet to fully rollout to all the app nodes.